src/features/call/pages/CallPage.tsx

```diff
@@ -66,7 +66,7 @@ const CallPage: FC = () => {
           </Box>
         }
       >
-        <Call />
+        {canInitialize ? <Call /> : null}
```

Comment:
In this codebase the more typical way to do this would be:

```suggestion
        {canInitialize && <Call />}
```

Reply:
Good catch, don't know why I did it that way.